changelog/README.md

```diff
@@ -2,8 +2,10 @@
 
 <!-- Note: contrary to 3.x, insert new entries *first* in their section -->
 
+
 ### 4.2.0 (in progress)
 
+- [improvement] JAVA-2307: Improve @Select and @Delete by not requiring full primary key
```

Comment:
Tip: place all words starting with `@` within backticks here and in commit messages, to avoid having Github consider that these are user handles.

Reply:
Thanks, I'll amend the commit when merging. It looks like github doesn't hijack markdown in the same way, but agree that we shoulds use backticks here as well.